### PR TITLE
[monaco] Fix keybindings on os x

### PR DESCRIPTION
**Pull request by svenefftinge**
Fixes theia-ide/theia#3834

Signed-off-by: Sven Efftinge <sven.efftinge@typefox.io>

### CODE REVIEW

**Comment by svenefftinge** [comment]
@kittaakos could you check on windows as well?

File: packages/monaco/src/browser/monaco-keybinding.ts [hunk 1]
```diff
@@ -79,7 +79,7 @@ export class MonacoKeybindingContribution implements KeybindingContribution {
             first: Key.getKey(monaco2BrowserKeyCode(keyCode & 255)),
             modifiers: []
         };
-        if (keybinding.ctrlKey) {
+        if (keybinding.metaKey) {
```
Comment (kittaakos) [inline packages/monaco/src/browser/monaco-keybinding.ts]: I think with this change, we never add Ctrl to the sequence in Windows... I am verifying it anyways

Comment (svenefftinge) [reply packages/monaco/src/browser/monaco-keybinding.ts]: We rely on the mapping logic from monaco:
https://github.com/Microsoft/vscode/blob/master/src/vs/base/common/keyCodes.ts#L428

Comment (kittaakos) [reply packages/monaco/src/browser/monaco-keybinding.ts]: The logic should be something like this 
 - `IF OS === macOS AND isMeta AND isCtrl` then it is a `CtrlCmd`
 - `IF OS === macOS AND !isMega AND isCtrl` the it is a `MacCtrl` (this should not do anything on Windows)
 - `IF OS !== macOS AND isCtrl` then it is a `CtrlCmd`

Comment (kittaakos) [reply packages/monaco/src/browser/monaco-keybinding.ts]: I see. So it is already handled by monaco. Thanks for the clarification.

**Comment by kittaakos** [comment]
Space does not work anymore in Windows. It always shows the CA instead:

screencast 2018-12-14 11-39-54

**Comment by svenefftinge** [comment]
oh, how convenient :-)

**Pull request merged**